src/main/java/com/puppycrawl/tools/checkstyle/Main.java

```diff
@@ -723,35 +723,34 @@ private static final class CliOptions {
 
         /** Option that controls whether to print the AST of the file. */
         @Option(names = {"-t", "--tree"},
-                description = "Prints Abstract Syntax Tree(AST) of the checked file. The option "
-                        + "cannot be used other options and requires exactly one file to run on "
-                        + "to be specified.")
+                description = "This option is used to display the Abstract Syntax Tree (AST) "
```

Comment:
I question making so many options say "This option...". We don't do this for every option, and for the ones which were changed, we don't do this for `javadocTree` which seems would follow the pattern of the others which were changed.

It seems to me every option should start with a plural verb. IE "Prints", "Excludes", "Checks", etc... This is almost how we did our XDoc/Javadoc documentation for properties during the first alignment rewrite.
https://checkstyle.org/checks/annotation/annotationlocation.html#Properties

Reply:
Ignore this.